### PR TITLE
Feature/create send button

### PR DESCRIPTION
**Pull request by pfbarranco**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -90,29 +91,80 @@ function printScore() {
...
+
+let sendButton = document.getElementById("sendButton");
+
+/* Send button is disabled by default */
+
+sendButton.disabled = true;
+
+/* Send button is enabled after clicking on evaluate buttons */
+
+evaluateButton1.addEventListener("click", function () {
```
Comment (carlosjfcasero) [inline src/app.js]: You have defined two "click" event listener. You could inline in one function

File: src/app.js [hunk 2]
```diff
@@ -90,29 +91,80 @@ function printScore() {
...
+    sendButton.disabled = false;
+});
+evaluateButton2.addEventListener("click", function () {
```
Comment (carlosjfcasero) [inline src/app.js]: You have defined two "click" event listener. You could inline in one function

File: src/app.js [hunk 3]
```diff
@@ -90,29 +91,80 @@ function printScore() {
...
+});
+
+/* Confirmation message pops up after clicking on send button */
+
+sendButton.addEventListener("click", function () {
+    askForConfirmationMessage();
+});
+
+function askForConfirmationMessage() {
+    let text = "Do you really want to send your evaluations?";
```
Comment (carlosjfcasero) [inline src/app.js]: Maybe you can use "const" instead of "let" here

File: src/app.js [hunk 4]
```diff
@@ -90,29 +91,80 @@ function printScore() {
...
+    if (confirm(text) == true) {
+        confirmationMessage();
+    }
+};
+
+function confirmationMessage() {
+    let text = "Evaluations properly sent!";
```
Comment (carlosjfcasero) [inline src/app.js]: Maybe you can use "const" instead of "let" here

File: test/app.test.js [hunk 5]
```diff
@@ -151,5 +152,86 @@ describe('Main page test', () => {
...
+            assert.isTrue(sendButton.disabled)
+        })
+        it("Send button is enabled after clicking on Evaluate", () => {
+            let app = require("../src/app");
+            let evaluateButton1 = document.getElementById('evaluateButton1');
+            let evaluateButton2 = document.getElementById("evaluateButton2");
+
+            var eventDispatched = evaluateButton1.dispatchEvent(new Event('click'))
+            assert.isTrue(eventDispatched);
+            var eventDispatched = evaluateButton2.dispatchEvent(new Event('click'))
```
Comment (carlosjfcasero) [inline test/app.test.js]: Do you need bot evaluateButton1 and evaluateButton2 to be clicked?

**Review by carlosjfcasero — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 3 above)
Comment (pfbarranco) [reply src/app.js]: Done

File: src/app.js (hunk 4 above)
Comment (pfbarranco) [reply src/app.js]: Done

File: test/app.test.js (hunk 5 above)
Comment (pfbarranco) [reply test/app.test.js]: Deleted nº2

File: src/app.js (hunk 1 above)
Comment (pfbarranco) [reply src/app.js]: Done

File: src/app.js (hunk 2 above)
Comment (pfbarranco) [reply src/app.js]: Done

**Review by carlosjfcasero — APPROVED** [review]

**Pull request merged**